src/Features/Core/Portable/DocumentHighlighting/IDocumentHighlightsService.cs

```diff
@@ -20,7 +20,7 @@ internal enum HighlightSpanKind
     }
 
     [DataContract]
-    internal readonly struct HighlightSpan
+    internal readonly record struct HighlightSpan
```

Comment:
❓ Why is this one changing?

Reply:
➡️ This type isn't used as a dictionary key, but rather as a dictionary value in a type that happens to create a set from the values.